static/js/stream_create.js

```diff
@@ -292,7 +292,6 @@ exports.show_new_stream_modal = function () {
             }
         });
 
-        update_announce_stream_state();
```

Comment:
Can you explain in the commit message your reasoning for why this is unnecessary?